src/node_api_types.h

```diff
@@ -67,6 +67,7 @@ typedef enum {
   napi_generic_failure,
   napi_pending_exception,
   napi_cancelled,
+  napi_escape_called_twice,
```

Comment:
A `#define` has the same issue: its value is fixed at compile time.  If there is no real use case for `napi_status_last`, I'd remove it.